src/invoices/InvoiceForm/InvoiceForm.js

```diff
@@ -466,6 +468,9 @@ const InvoiceForm = ({
                           type="number"
                           readOnly={!isExchangeRateEnabled}
                           tooltipText={tooltipTextExchangeRate}
+                          required={isExchangeRateRequired}
+                          validate={isExchangeRateRequired ? validateRequired : undefined}
+                          key={isExchangeRateRequired ? 1 : 0}
```

Comment:
really required?

Reply:
@NikitaSedyx there is a buggy behavior in rff, if validation requirement is dynamic we have to put key so component is rerendered. https://final-form.org/docs/react-final-form/types/FieldProps#validate 

> ⚠️ IMPORTANT ⚠️ – By default, in order to allow inline fat-arrow validation functions, the field will not rerender if you change your validation function to an alternate function that has a different behavior. If you need your field to rerender with a new validation function, you will need to update another prop on the Field, such as key. See the following sandbox for an example: